recipe/meta.yaml

```diff
@@ -22,17 +22,17 @@ requirements:
     - ipython >=5.0
     - jupyter_client
     - pip
-    - python
+    - python >=3.4
```

Comment:
Given it's no longer tested on 3.4, shouldn't this be 3.5?

Reply:
Surely it should match whatever setup.py says. i.e. what it can be installed with, rather than what it should be installed with.